### DISCUSSION TITLE
Restore Purchases does not work...

### DISCUSSION THREAD

**Discussion by HumanDigital** [rsayfutdinov/Pikka#33] [General]
Hi Ruslan
I recently bought the Pro version from the Appstore and installed it on mine and my girlfriend's Mac (we are also merged as a "family" in the Appstore).
However, when I click on "Restore Purchases" in Pikka on my girlfriend's MacBook, the message "Successfully restored all purchases" appears but the navigation item "Get Pikka PRO..." does not disappear....
Can you help me/us with this?
Many thanks and greetings
Christoph

**Comment by rsayfutdinov** [comment]
Hi Christoph,

I can't say exactly why is not working.
But I just generated a promo code for you, so you can use it on the second MacBook.
Here it is: 6TY3KL6WW3FL

Regards,
Ruslan

**Reply by HumanDigital** [reply]
Hello Ruslan
Thank you for your kindness and the promo code. Unfortunately, the promo code did not work ... but that is probably my fault and so the license was charged again ... but that's ok :-)
Many thanks and kind regards
Christoph

**Reply by JVPartanen** [reply]
I have the same problem. Have been using Pikka for long time and now when I press "restore purchases", it says "Successfully restored all purchases" but still it is unlicensed. 😞

**Reply by JVPartanen** [reply]
I started to use Sip app instead, problem solved

**Comment by spiral2k** [comment]
Hey, I have this problem too, bought Pikka a couple of years ago but I keep seeing the "Upgrade to Pro" pop up every time, what can i do?

**Reply by rsayfutdinov** [reply]
Here is a promo code for you: NAKK9LLN3M4F
You can redeem it in the mac app store.

Regards,
Ruslan

**Reply by spiral2k** [reply]
works perfect, thank you Ruslan

**Comment by Trailingslashes** [comment]
@rsayfutdinov   My purchase restore doesn't work either. However, I did buy it 2 years ago.

**Reply by rsayfutdinov** [reply]
Here is a promo code for you
**KE63A3APPXXF**

**Reply by Trailingslashes** [reply]
Thanks that worked!

**Comment by RBHannam** [comment]
Hi,
I'm having a similar problem. I migrated from a McPro to a Mac Mini and Pikka came too. Unfortunately, the licence number I have won't work on the new machine. I'm happy to uninstall from the old one if that would help? Grateful for some advice. Thanks.

**Comment by AitchHubber** [comment]
I have this issue too @rsayfutdinov . I've bought the pro licence. I haven't used it for a little while, came to use it and it wont restore the purchase. Ypur assistance would be appreciated. (britishsoldier@icloud.com)
